plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/custom/deserializer/BeanMediatorExtFactory.java

```diff
@@ -67,6 +67,9 @@ public Mediator createSpecificMediator(OMElement elem, Properties properties) {
                 // ignore
             }
         }
+        
```

Comment:
Unnecessary new line. Shall we remove that in other places too?

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/1063